.github/workflows/deploy-pages.yml

```diff
@@ -0,0 +1,55 @@
+name: deploy-pages
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: pages
+  cancel-in-progress: false
+
+jobs:
+  build-docs:
+    uses: ./.github/workflows/doc.yml
+  build-site:
+    runs-on: ubuntu-latest
+    needs: build-docs
+    steps:
+      - name: Check out Celeritas doc base
+        uses: actions/checkout@v4
+        with:
+          ref: doc/gh-pages-base
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+      - name: Download doc and user artifacts
+        uses: actions/download-artifact@v4
+      - name: Rename artifacts
+        run: |
+          mv doc-dev dev
+          mv doc-user user
+      - name: Build with Jekyll
+        uses: actions/jekyll-build-pages@v1
+        with:
+          source: ./
+          destination: ./_site
```

Comment:
Out of curiosity (our source does not seem to have mention of `_site`), whose convention is this name from? (humm https://github.com/actions/jekyll-build-pages seems to say that this is the default value anyway).

Reply:
This was part of the default build template github provided...

Reply:
That ends up confusing me :).  By specifying it, one would assume it is not the default and then the next step look 'weird' (needs lookup of the doc to understand whether it works or not.

I would recommend to either note explicitly it is the default (possibly even not set it, just say where it will be created) **or** also add the corresponding setting in the upload-pages-artifact part.